.github/workflows/test-code.yml

```
@@ -20,7 +20,7 @@ jobs:
           - stable
     runs-on:                       ubuntu-18.04
     container:
-      image:                       paritytech/ci-linux:e79b7a7e-20200827
+      image:                       paritytech/ci-linux:4843e4d0-20201231
```

you can safely use `production` tag, it's tested against substrate every night.
```suggestion
      image:                       paritytech/ci-linux:production
```